### DISCUSSION TITLE
Is there a way to generate book indices for HTML?

### DISCUSSION THREAD

**Discussion by francojc** [quarto-dev/quarto-cli#5579] [Q&A]
I'm preparing a book and I will be adding an index page. In the PDF version I can use [`makeidx` with LaTeX](https://www.overleaf.com/learn/latex/Indices). Does Quarto have built in functionality to create indices for HTML? If not, what options are there for producing something similar to `makeidx` for HTML output?

**Comment by francojc** [comment]
I just stumbled across this issue #3849 which is getting at the same question. That Q&A thread is also unanswered and has gone cold. Any suggestions? JavaScript? Lua filter? I'm open.

**Reply by mcanouil** [reply]
The thread you mentioned has an answer about a filter that could possibly do the job.
Now, someone has to make it work.

Otherwise, you'll have to wait quite some time before this kind of "niche" feature comes into Quarto.

**Reply by cscheid** [reply]
This kind of feature is on our radar. It's at least a few months out, but it's the same general feature as a glossary, and we'll totally do it.

Only so many hours in the day, though...

**Comment by mcanouil** [comment]
I am going to close this (as a duplicate of #3849) in favour of the feature request discussion to avoid dispersing the discussion over multiple threads.

Please continue on #3849
